python/grass/app/runtime.py

```diff
@@ -17,12 +17,48 @@
 import subprocess
 import sys
 
+import grass.app.resource_paths
```

Comment:
`import .resource_paths` should work here.

Reply:
That gives:

```
    import .resource_paths
           ^
SyntaxError: invalid syntax
```
but,

```py
from . import resource_paths
```

works. I'm not sure whether that would be better though, at this point `sys.path.append(GRASS_PYDIR)` should have been set.

Reply:
Right, it's the from syntax. While I don't know if there are any import procedure differences, the dot import seems to correctly represent the "local" situation.

Reply:
Changed to dot import.
(We wouldn't load runtime.py without having the grass module in path anyway).